edx_proctoring/tests/test_api.py

```diff
@@ -1648,3 +1648,19 @@ def test_declined_prerequisites(self, content_id,
         self.assertEqual(len(results['failed_prerequisites']), expected_len_failed_prerequisites)
         self.assertEqual(len(results['pending_prerequisites']), expected_len_pending_prerequisites)
         self.assertEqual(len(results['declined_prerequisites']), expected_len_declined_prerequisites)
+
+    def test_summary_without_credit_state(self):
+        """
+        Test that status summary is None for unenrolled user
+        that does not have a corresponding credit state.
+        """
```

Comment:
Sorry I didn't get it. What summary we are discussing here? and corresponding credit state?